.github/workflows/releases.yaml

```diff
@@ -23,7 +23,9 @@ jobs:
         env:
           USER: ${{ secrets.QUAY_USERNAME }}
           PASS: ${{ secrets.QUAY_PASSWORD }}
-        run: docker login -u "$USER" -p "$PASS" quay.io && make push
+        run: |
+          docker login -u "$USER" -p "$PASS" quay.io 
+          make push && make release
```

Comment:
Notice here @matejvasek that, when the push is for a versioned release, it pushes all tags and then additionally tags the :latest.